DOCKER.md

```diff
@@ -33,10 +33,14 @@ root@dec4c0f12478:/src# /app/bin/jenkinsfile-runner -w /tmp/jenkins -p /usr/shar
 ```
 
 ## Debug
-In case you want to debug Jenkinsfile Runner the recommendation is to use the image built following the steps mentioned in the section above.
+In case you want to debug Jenkinsfile Runner, you need to use the Vanilla Docker image built following the steps mentioned in the section above.
 
-Then, set the `DEBUG` environment variable and expose the port where to connect the remote debug
+Then, set the `DEBUG` environment variable and expose the port where to connect the remote debug. Note Jenkinsfile Runner itself
+considers 5005 as debugging port but you can map such port to whatever value you prefer through Docker port mapping.
 
 ```bash
 docker run --rm -e DEBUG=true -p 5005:5005 -v $PWD/test:/workspace jenkinsfile-runner:my-production-jenkins
 ```
+
+In case you are having issues when the Docker image is generated in another way (for example through [Custom WAR Packager](https://github.com/jenkinsci/custom-war-packager/)),
+you can simulate the configuration there (plugin versions, WAR file, ...) through the Vanilla image and then debug.
```

Comment:
I would recommend just adding the JAVA_OPTS command sample here

Reply:
In theory you'll want to debug when you are developing Jenkinsfile Runner or having any issues. Simulating a configuration (for example a Docker image generated through Custom War Packager) should be possible through the plugins.txt file and some configuration, so no need to use the JAVA_OPTS option I think (or at least mentioning it here).

Reply:
```suggestion
you can directly pass `JAVA_OPTS` to the Docker run arguments:

```bash
docker run --rm -e JAVA_OPTS='-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005' -p 5005:5005 -v $PWD/test:/workspace jenkinsfile-runner:my-production-jenkins
```

```

Reply:
Rendered like 💩 , but I hope it represents what I meant